doc/source/whatsnew/v1.3.0.rst

```diff
@@ -33,17 +33,12 @@ For example:
         storage_options=headers
     )
 
-.. _whatsnew_130.window_method_table:
```

Comment:
this section appears under the https://pandas.pydata.org/pandas-docs/dev/whatsnew/v1.3.0.html#custom-http-s-headers-when-reading-csv-or-json-files section when rendered so moved to Other enhancements